ethereum-rpc/src/constructors/express/routes/api/v1/uniswap.js

```diff
@@ -0,0 +1,512 @@
+const router = require("express").Router();
+const ethers = require('ethers');
+const { ChainId, WETH, Fetcher, Trade, Route, TokenAmount, TradeType, Percent } = require('@uniswap/sdk')
+const { query, param, validationResult } = require("express-validator");
+
+
+module.exports = ({
+  models,
+  ethereum: { web3, buildContract, compiler, abi },
+  ...context
+}) => {
+
+  router.post(
+    "/:address/swap/eth", [
+    param("address").trim().isString(),
+    query("tokenFrom").trim().isIn(["eth"]),
+    query("tokenTo").trim().isIn(["dai", "mkr", "uni"]),
+    query("privateKey").trim().isString(),
+  ],
+    async (req, res, next) => {
+      try {
+        const errors = validationResult(req);
+        if (!errors.isEmpty()) {
+          return res.status(400).json({ errors: errors.array() });
+        }
+        const { address = "*" } = req.params;
+        const { privateKey = "", amount = "", tokenFrom = "", tokenTo = "" } = { ...req.query };
+        const tokenType = tokenTo.toString().toUpperCase();
+        const routerAbi = abi.UNISWAP_ROUTER.abi
+        const chainIds = ChainId.RINKEBY;
+
+        const tokenAddressRinkebyDai = '0xc7ad46e0b8a400bb3c915120d284aafba8fc4735'
+        const tokenAddressRinkebyUniswap = '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984';
+        const tokenAddressRinkebyMaker = '0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85';
+        const tokenAddressRinkebyRouter = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
```

Comment:
These will go in a map by network

Reply:
this address will be taken as inputs in the coming version